### DISCUSSION TITLE
part-ii/2018-25/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#132] [Question comments]
Part II 2018 Automata and Formal Languages: Paper 1, Section I, G 

https://questions.tripos.org/part-ii/2018-25/

**Comment by artie2000** [comment]
(i) No (consider $a^nb^na^nb^n$)
<details>
<summary>(ii) Yes:</summary>

$S\to XY,\ X\to a^4Xb^3\mid\varepsilon,\ Y\to c^5Yd^2\mid\varepsilon$

</details>

(iii) No
